### PR TITLE
Redid Browse Screen.

### PR DESCRIPTION
**Pull request by rauchp**
- Standerized the browse screen header, now all headers in the app should be the same
- Cleaned up BrowseScreen a little.
- Redid SportsCardNew, including de-asseting the drop shadow 
- Waiting on team to finalize what the UnknownBanner image is going to be.
new_Brwose

### CODE REVIEW

**Comment by rauchp** [comment]
Screen Shot 2019-12-28 at 11 59 30 AM

Pixel Tweaked, this is what final looks like so far

File: civilapp/src/browse/RoomsView.js [hunk 1]
```diff
@@ -1,27 +1,13 @@
...
-  StyleSheet,
-  View,
-  ScrollView,
-  RefreshControl,
-  FlatList,
-} from 'react-native'
+import { Dimensions, StyleSheet, View, ScrollView, RefreshControl, FlatList } from 'react-native'
 import AsyncStorage from '@react-native-community/async-storage'
 import SportsCardNew from './common/SportsCardNew'
 import { TeamDisplayNameMap } from '../sports/utils/TeamNameMap'
```
Comment (EphemeralX) [inline civilapp/src/browse/RoomsView.js]: nit: import order

File: civilapp/src/browse/common/SportsCardNew.js [hunk 2]
```diff
@@ -1,38 +1,27 @@
...
-  sbCreateParticipantListQuery,
-  sbGetParticipantList,
-} from '../../ChatUtils'
-
-const { height, width } = Dimensions.get('window')
+import { View, Text, TouchableOpacity, ImageBackground, StyleSheet, Dimensions } from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+import { SuperTextWithShadow } from '../../components/SuperText'
+import { getBanner } from '../BannerUtil'
+import { sbCreateParticipantListQuery, sbGetParticipantList } from '../../ChatUtils'
```
Comment (EphemeralX) [inline civilapp/src/browse/common/SportsCardNew.js]: nit: import order

File: civilapp/src/browse/common/SportsCardNew.js [hunk 3]
```diff
@@ -41,174 +30,141 @@ class SportsCardNew extends Component {
...
+    </View>
+  )
+
+  renderGradient = () => (
+    <LinearGradient
+    style={styles.gradient}
+    colors={colors}
+  />
+  )
+  renderTitle = () => (
```
Comment (EphemeralX) [inline civilapp/src/browse/common/SportsCardNew.js]: nit: new line between functions

File: civilapp/src/browse/common/SportsCardNew.js [hunk 4]
```diff
@@ -41,174 +30,141 @@ class SportsCardNew extends Component {
...
   render() {
-    const { size, onlineCount } = this.state
-    const counterWidth = 11 + onlineCount.toString().length + '%'
+    const { size } = this.state
+    const { roomPhoto } = this.props
+    const widthStyle = { width: size * 0.915 }
+
+    let banner
+    if (roomPhoto) banner = getBanner(roomPhoto)
+    else banner = UnknownBanner
```
Comment (EphemeralX) [inline civilapp/src/browse/common/SportsCardNew.js]: `const banner = roomPhoto ? getBanner(roomPhoto) : UnknownBanner`

File: civilapp/src/browse/common/SportsCardNew.js (hunk 2 above)
Comment (rauchp) [reply civilapp/src/browse/common/SportsCardNew.js]: what's wrong with this order?

**Pull request merged**